src/test/ml-modules/root/test/suites/auditing/property-history.xqy

```diff
@@ -14,8 +14,19 @@ declare variable $property-list := (
 
 declare option xdmp:mapping "false";
 
+let $assertions := ()
+
 let $merged-uri := cts:uris((), "limit=1", cts:collection-query($const:MERGED-COLL))
 let $actual as map:map := history:property-history($merged-uri)
-return (
+let $assertions := (
+  $assertions,
   test:assert-same-values($property-list, map:keys($actual))
 )
+
+(: normalize-value-for-tracing should not return empty for text nodes :)
+let $assertions := (
+  $assertions,
+  test:assert-equal(history:normalize-value-for-tracing(text{'textValue'}), 'textValue')
```

Comment:
`assert-equal` has the expected value first, then the actual. @patrickmcelwee please reverse these two, and then I'll merge the PR.

Reply:
Thanks Dave ... I made that change and rebased it into the commit.